Dockerfile

```diff
@@ -72,6 +72,10 @@ WORKDIR /tmp/nemo
 COPY requirements .
 RUN for f in $(ls requirements*.txt); do pip3 install --disable-pip-version-check --no-cache-dir -r $f; done
 
+# install flash attention dependencies
+RUN pip install flash-attn
+RUN pip install triton==2.0.0.dev20221202
```

Comment:
Does this need to be pinned?

Reply:
FA has fixed triton version currently. I have tried other version which will raise errors.
https://github.com/HazyResearch/flash-attention/blob/main/flash_attn/flash_attn_triton.py#L3